### PR TITLE
Update to Contribution Process

### PR DESCRIPTION
**Pull request by okorshenko**
Updated information about target branches for public Pull Requests according to the changes in Public Pull Requests processing flow:
- [ ] added information about new supported branches 2.1-develop, 2.2-develop and 2.3-develop
- [ ] removed the reference to develop branch since the branch was removed and replaced with 2.3-develop (ad for today)
- [ ] Replaced references to Magento 2 Core Team with Magento 2 Community Maintainers Team to reflect real process
- [ ] Updated information about GitHub labels for the pull requests and issues
- [ ] Updated all URL to `develop` branch and replaced with `2.3-develop`

### CODE REVIEW

File: guides/v2.0/contributor-guide/contributing.md [hunk 1]
```diff
@@ -37,9 +37,9 @@ This contribution model has contributors maintaining their own copy of the forke
 Contributions can take the form of new components or features, changes to existing features, tests, documentation (such as developer guides, user guides, examples, or specifications), bug fixes, optimizations, or just good suggestions.
 
 
-The Magento 2 development team reviews all issues and contributions submitted by the community of developers in a first-in, first-out basis. During the review we might require clarifications from the contributor. If there is no response from the contributor for two weeks, the issue is closed.
```
Comment (jfrontain) [inline guides/v2.0/contributor-guide/contributing.md]: change "in" to "on"

File: guides/v2.0/contributor-guide/contributing.md [hunk 2]
```diff
@@ -98,11 +98,11 @@ To create a pull request do the following:
...
 2. In your repository, click **Pull requests** on the right, and then click **New pull request**: <br><img src="{{ site.baseurl }}common/images/pr.png" target="_blank">
-3. Ensure that you are creating a PR to the `magento:develop` branch. We accept PRs to this branch only.
+3. Ensure that you are creating a PR to the one of following  branches: `magento:2.3-develop`, `magento:2.2-develop` or `magento:2.1-develop` branch. We accept PRs to these branches only.
 4. Review the changes, then click **Create pull request**. Fill out the form, and click **Create pull request** again to submit the PR&mdash;that’s it!
 
 
-After submitting your PR, you can head over to the Magento 2 repository’s <a href="https://github.com/magento/magento2/pulls?q=is%3Aopen+is%3Apr" target="_blank">Pull Requests panel</a> to see your PR along with the others. Your PR undergoes automated testing, and if it passes, the core team considers it for inclusion in the Magento 2 core. If some tests fail, please make the corresponding corrections in your code.
```
Comment (jfrontain) [inline guides/v2.0/contributor-guide/contributing.md]: remove "the" from "along with the others"

File: guides/v2.0/contributor-guide/contributing.md [hunk 3]
```diff
@@ -137,55 +137,75 @@ Refer to the following table for a description of each label. These labels are a
...
-       <tr>
-         <td><img src="{{ site.baseurl }}common/images/github_DOC.png" alt="the Doc button"/></td>
-         <td>Affects Documentation domain.</td>
+      <tr>
+         <td><img src="{{ site.baseurl }}common/images/github_accept.png" alt="the Accept button"/></td>
+         <td>The pull request has been accepted and will be merged into mainline code.</td>
       </tr>
       <tr>
-         <td><img src="{{ site.baseurl }}common/images/github_PROD.png" alt="the PROD button"/></td>
-         <td>Affects the Product team (mostly feature requests or business logic change).</td>
```
Comment (jfrontain) [inline guides/v2.0/contributor-guide/contributing.md]: business logic changes

File: guides/v2.0/contributor-guide/contributing.md (hunk 1 above)
Comment (okorshenko) [reply guides/v2.0/contributor-guide/contributing.md]: the text in red is deleted. What should I change?

File: guides/v2.0/contributor-guide/contributing.md (hunk 3 above)
Comment (okorshenko) [reply guides/v2.0/contributor-guide/contributing.md]: this text is deleted

**Comment by jfrontain** [comment]
Oleksii, my mistake: Ignore my previous comments, which applied to deleted text.

File: guides/v2.0/contributor-guide/contributing.md (hunk 2 above)
Comment (okorshenko) [reply guides/v2.0/contributor-guide/contributing.md]: fixed

**Comment by orlangur** [comment]
Great job on making documentation links great again 👍 

> Updated all URL to `develop` branch and replaced with `2.3-develop`

I have a little concern regarding this. Besides removing `develop` branch in favor of `2.3-develop` we did one more thing: changed default repo branch to `2.2-develop` (current stable VS WIP release like it was before). As contributors will normally start work from `2.2-develop` branch, I believe it is a better replacement for `develop` in most if not all cases.

Another thing is file references, especially on particular code lines. It is better to use stable tags corresponding to guide version in such case (like `guides/2.2` should refer to `2.2` and not `2.2-develop` or some other version). But it can be changed in separate improvement.

Would be nice to squash changes before merging and thanks again for your efforts, really appreciate having a doc we can actually refer to during PR processing.

**Comment by okorshenko** [comment]
Hi @orlangur thank you for feedback. I will take this into account.

**Pull request merged**